src/lib/commandBarConfigs/modelingCommandConfig.ts

```diff
@@ -368,7 +368,6 @@ export const modelingMachineCommandConfig: StateMachineCommandSetConfig<
   Revolve: {
     description: 'Create a 3D body by rotating a sketch region about an axis.',
     icon: 'revolve',
-    status: 'development',
```

Comment:
@nadr0 I think we'd want to remove the warningMessage prop below as well?

Reply:
Yes, lemme do that.